app/controllers/api_controller/parameters.rb

```diff
@@ -28,26 +28,38 @@ def hash_fetch(hash, element, default = {})
     def filter_param(klass)
       return nil if params['filter'].blank?
 
-      operators = {"!=" => {:default => :not_eq, :str => :does_not_match},
+      operators = {"!=" => {:default => :not_eq, :str => :does_not_match, :ruby => :!=},
                    "<=" => {:default => :lteq},
                    ">=" => {:default => :gteq},
                    "<"  => {:default => :lt},
                    ">"  => {:default => :gt},
-                   "="  => {:default => :eq, :str => :matches}
```

Comment:
We could, but I'm concerned with the PR getting out of scope, as I'd like to test that stuff quite thoroughly and it would take us away from the original use case defined in the BZ. @gtanzillo added this card here: https://trello.com/c/iWuQq8cK/160-api-support-full-filtering that I'd like to tackle in another PR, which would add full support for stuff like this.

I'll update the PR title and commit message to better reflect the scope of this.